security/integration/utils/kubernetes.go

```diff
@@ -103,12 +107,14 @@ func CreateService(clientset kubernetes.Interface, namespace string, name string
 	return clientset.CoreV1().Services(namespace).Get(name, metav1.GetOptions{})
 }
 
+// DeleteService takes identity information of a service and deletes the service.
```

Comment:
Remove "takes identity information of a service"?

Reply:
Done. Thanks!